starter-code/clue.js

```diff
@@ -1,104 +1,149 @@
+  occupation: "Scientist"
+};
+
+  var profPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  color: "purple",
+  description: "Billionare video game designer",
+  age: 22,
+  image: "https://metrouk2.files.wordpress.com/2016/07/professor-plum.jpg",
+  occupation: "Designer"
+};
+
+  var missScarlet = {
+  firstName: "Kasandra",
+  lastName: "Scarlet",
+  color: "red",
+  description: "She is an A-list movie star with a dark past",
   age: 31,
-  image: https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg,
-  occupation: Actor
-
- Mrs Peacock
-  first name: Eleanor
-  last name: Peacock
-  color: blue
-  description: She is from a wealthy family and uses her status and money to earn popularity
-  age: 36
-  image: https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg
-  occupation: Socialité
-
- Mr Mustard
-  first name: Jack
-  last name: Mustard
-  color: yellow
-  description: He is a former football player who tries to get by on his former glory
-  age: 62
-  image: https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg
-  occupation: Retired Football player
-
----- Weapons ----
-
-Rope - Weight: 10
-Knife - Weight: 8
-Candlestick - Weight: 2
-Dumbbell - Weight: 30
-Poison - Weight: 2
-Axe - Weight: 15
-Bat - Weight: 13
-Trophy - Weight: 25
-Pistol - Weight: 20
-
-
----- Rooms ----
-Dinning Room
-Conservatory
-Kitchen
-Study
-Library
-Billiard Room
-Lounge
-Ballroom
-Hall
-Spa
-Living Room
-Observatory
-Theater
-Guest House
-Patio
-
-*/
+  image: "https://metrouk2.files.wordpress.com/2016/07/miss-scarlett.jpg",
+  occupation: "Actor"
+};
+
+ var mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "She is from a wealthy family and uses her status and money to earn popularity",
+  age: 36,
+  image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+  occupation: "Socialité"
+};
+ var mrMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "He is a former football player who tries to get by on his former glory",
+  age: 62,
+  image: "https://metrouk2.files.wordpress.com/2016/07/colonel-mustard.jpg",
+  occupation: "Retired Football player"
+};
+
+var rope = {
+  weapon:"rope",
+  weight: 10
+};
+var knife = {
+  weapon:"knife",
+  weight: 8
+};
+var candlestick = {
+  weapon:"candlestick",
+  weight: 2
+};
+var dumbbell = {
+  weapon:"dumbbell",
+  weight: 30
+};
+var poison = {
+  weapon:"poison",
+  weight: 2
+};
+var axe = {
+  weapon:"axe",
+  weight: 15
+};
+var bat = {
+  weapon:"bat",
+  weight: 13
+};
+var trophy = {
+  weapon:"trophy",
+  weight: 25
+};
+var pistol = {
+  weapon:"pistol",
+  weight: 20
+};
+
+
+//---- Rooms ----
+var stackCardRoom = [
+"Dinning Room",
+"Conservatory",
+"Kitchen",
+"Study",
+"Library",
+"Billiard Room",
+"Lounge",
+"Ballroom",
+"Hall",
+"Spa",
+"Living Room",
+"Observatory",
+"Theater",
+"Guest House",
+"Patio"
+];
```

Comment:
It would have made more sense to use objects instead of strings, but this works too.